### PR TITLE
Create Team Controller

### PR DESCRIPTION
**Pull request by MikolajWojewoda**

### CODE REVIEW

File: src/FantasyFun.API/Controllers/TeamsController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace FantasyFun.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class TeamsController : ControllerBase
+    {
+        [HttpGet]
+        [Route("{id}")]
+        public async Task<ActionResult<TeamName>> GetAnyTeam(long id)
```
Comment (ficmichal) [inline src/FantasyFun.API/Controllers/TeamsController.cs]: Not valid method name - I suggest "GetTeamById"

File: src/FantasyFun.API/Controllers/TeamsController.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        {
+            var db = new FootballDbContext();
+
+            var anyTeam = await db.Teams
```
Comment (ficmichal) [inline src/FantasyFun.API/Controllers/TeamsController.cs]: Not valid variable name - I suggest "team"

File: src/FantasyFun.API/Models/Team.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿namespace FantasyFun.API.Models
+{
+    public class Team
+    {
+        public long Id { get; set; }
+        public long FifaApiId { get; set; } 
+        public long ApiID { get; set; }
```
Comment (ficmichal) [inline src/FantasyFun.API/Models/Team.cs]: Maybe better is stick to one convention - Id not ID

File: src/FantasyFun.API/ViewModel/TeamName.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿namespace FantasyFun.API.ViewModel
+{
+    public class TeamName
```
Comment (ficmichal) [inline src/FantasyFun.API/ViewModel/TeamName.cs:3]: I recommend to use readonly properties here (only get, not set) and create a constructor

**Review by ficmichal — CHANGES_REQUESTED** [review]
PR review

File: src/FantasyFun.API/Models/Team.cs (hunk 3 above)
Comment (MikolajWojewoda) [reply src/FantasyFun.API/Models/Team.cs]: ok

**Review by ficmichal — APPROVED** [review]

**Pull request merged**